includes/rcon.sh

```diff
@@ -1,61 +1,61 @@
 # shellcheck disable=SC2148
 
 function save_and_shutdown_server() {
-    rconcli 'broadcast Server-shutdown-was-requested-init-saving'
+    rconcli 'broadcast Server shutdown was requested init saving'
     rconcli 'save'
-    rconcli 'broadcast Done-saving-server-shuts-down-now'
+    rconcli 'broadcast Done saving server shuts down now'
 }
 
 function broadcast_automatic_restart() {
     time=$(date '+%H:%M:%S')
 
     for ((counter=1; counter<=15; counter++)); do
-        rconcli "broadcast ${time}-AUTOMATIC-RESTART-IN-$counter-MINUTES"
+        rconcli "broadcast ${time} AUTOMATIC RESTART IN $counter MINUTES"
```

Comment:
This is more of a recommendation than a hard requirement, but it'd be great if we could have timestamps (`${time}`) added to all broadcasts, possibly within the `rconcli.sh` script itself.

Reply:
Fair enough. I'll be happy to come up with a follow up change for proposing this.